### ISSUE TITLE
Test for WCAG technique G9:  Creating captions for live synchronized media

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#220]
http://www.w3.org/TR/WCAG20-TECHS/G9.html
The objective of this technique is to allow users who cannot hear to be able to access real-time synchronized media broadcasts. It is more difficult to create accurate real-time captions because there is little time to correct mistakes or to listen a second time or consult someone to be sure the words are accurately reproduced. It is also harder to simplify or paraphrase information if it is flowing too quickly.

Real-time typing text entry techniques exist using stenographic and rapid typing technologies. Re-voicing speech-to-text (where a person listens to speech and then carefully re-voices it into a computer trained to their speech) is used today for telephone relay services and may be used in the future for captioning. Eventually speech-to-text with correction will be possible.
## Applicablity

Live video with audio
## Procedure
1. Check that a procedure and policy are in place to ensure that captions are delivered in real-time.
## QUAIL
- This test should return a 'not Applicable' if no livestream situation is detected on the page, and a 'cantTell' if a livestream situation is detected. It won't return a pass or a fail as the detection of the captions iself are covered by other tests.
- As such this technique works as a precondition as the other techniques are testing if a caption is available.
- we need two different tests with detection mechanisms if a livestream is available. 
  We have 'always on' livestreams and
  livestreams that are triggered and has a text announcement if the livestream is not available.
  Let's start with an always on livestream. 
### Always available livestreams

A test that we can detect the type of streaming technique. Some video formats are exclusively used for livestreams. We need example code for that.
### Occasional available livestreams

We need some text snippets for this.
